doc/netplan-yaml.md

```diff
@@ -2,8 +2,54 @@
 title: "YAML configuration"
 ---
 
+## General configuration structure
```

Comment:
There's already a "General structure" topic in the manpage header... Maybe we should rename this to something different, like "Top-level configuration structure", to avoid confusion?